Client/mods/deathmatch/logic/luadefs/CLuaEngineDefs.h

```diff
@@ -48,7 +50,7 @@ class CLuaEngineDefs : public CLuaDefs
     LUA_DECLARE(EngineGetModelTextureNames);
     LUA_DECLARE(EngineGetVisibleTextureNames);
 
-    static bool EngineSetModelVisibleTime(const std::variant<std::string, unsigned int> variantModelID, char hourOn, char hourOff);
+    static bool EngineSetModelVisibleTime(lua_State* const luaVM, const std::variant<std::string, unsigned int> variantModelID, char hourOn, char hourOff);
```

Comment:
EngineGetModelVisibleTime. This shouldn't be here

```suggestion
    static bool EngineSetModelVisibleTime(const std::variant<std::string, unsigned int> variantModelID, char hourOn, char hourOff);
```

Reply:
I think ive squashed them when rearranging the commits yesterday.